### PR TITLE
 [util] Hide Intel for Kena: Bridge of Spirits to skip faulty water re…  …ndering path.

### PR DESCRIPTION
**Pull request by MickAlmighty**
… water rendering path.

### CODE REVIEW

**Comment by Blisto91** [comment]
Does it behave the same on Windows when it sees Intel?

**Comment by MickAlmighty** [comment]
Hi, this is the issue mentioned here: https://community.intel.com/t5/Graphics/Kena-Bridge-of-Spirits-Water-Glitch/td-p/1603888

There's also a yt video how the flicker looks like. 

The flicker is visible both on dx11 and dx12 but I captured a DX12 frame on Intel and replayed on NVidia and NV renders the same, hence this tells it is the game issue. The game detects Intel GPU and runs different path for water rendering which is buggy. It needs to be fixed by game devs, so at least it won't flicker when dxvk  is used :)

**Comment by MickAlmighty** [comment]
I tested it on Windows

**Comment by Blisto91** [comment]
I see the game doesn't ship amd_ags so it might not matter here but usually when wanting to just hide Intel it is preferred to use `dxgi.hideIntelGpu = True` so it doesn't touch the other vendors.
Though that would spoof as AMD not Nvidia.

**Comment by MickAlmighty** [comment]
Yeah, I've seen it used for BatmanAK where when it detects Intel it just closes the game.

I may try the dxgi.hideIntelGpu = True to see the outcome.

**Comment by MickAlmighty** [comment]
I changed it to hideIntelGpu = True, it's better because it tunes the dxvk behavior just for Intel.

Thank you for suggestion :)

**Comment by Blisto91** [comment]
👍 just need to change commit text to reflect that and I think it should be good.

**Review by Blisto91 — APPROVED** [review]

File: src/util/config/config.cpp [hunk 1]
```diff
@@ -485,6 +485,10 @@ namespace dxvk {
       { "d3d9.maxFrameRate",              "-1"      },
       { "dxgi.maxFrameRate",              "-1"      },
     }} },
+    /* Kena: Bridge of Spirits: intel water       * 
+      flickering issues                           */
+    { R"(\\Kena-Win64-Shipping\.exe$)", {{
+      { "dxgi.hideIntelGpu",                 "True" },
```
[the remark this answers is not in the document]
Comment (Blisto91) [reply src/util/config/config.cpp:491]: @MickAlmighty Friendly ping

**Comment by doitsujin** [comment]
Merged manually: 8c58bef9cff00023361db782b6a85c719eb1b7ac

**Pull request closed without merging**